### ISSUE TITLE
Do i need to install A7-TotLM-BG2EE mod?

### ISSUE THREAD

**Issue by zj11223344** [The-Gate-Project/IWD1_EET#12]
Dose this mod contain Trials of the Luremaster？Thanks for answer.

**Comment by 11jo** [comment]
Yes the mod contains Heart of Winter and Trials of the Luremaster.

**Comment by SiriusVI87** [comment]
Tipun's mods contain TotLM? I believe I have read that TotLM is not included in Tipun's mod. Can anyone confirm this?
